app/views/command-line.html

```diff
@@ -48,7 +48,7 @@ <h1 id="cli">Command Line Tools</h1>
                   <div ng-show="showSessionToken" class="alert alert-warning">
                     <span class="pficon pficon-warning-triangle-o" aria-hidden="true"></span>
                     <strong>A token is a form of a password.</strong>
-                    Do not share your API token.
+                    Do not share your API token. If your token may have been leaked or you otherwise want to reset it, please <a target="_blank" href="{{'token-request' | apiLink}}">Request a New API Token</a>. 
```

Comment:
Turns out that this statement isn't true. Token's aren't invalidated by fetching a new one.